src/models/Photo.js

```diff
@@ -23,22 +24,43 @@ class Photo extends Realm.Object {
     return localPhoto;
   }
 
-  static async resizeImageForUpload( path ) {
+  static async resizeImageForUpload( pathOrUri ) {
     const width = 2048;
     const { photoUploadPath } = Photo;
-
     await RNFS.mkdir( photoUploadPath );
+    let outFilename = pathOrUri.split( "/" ).slice( -1 ).pop( );
+
+    // If pathOrUri is an ios localIdentifier, make up a filename based on that
```

Comment:
Do we need to create a new filename here? The way I thought this was working is that resizeImage creates a new filepath in a temp directory, which we immediately save to a permanent filepath within `/photoUploads`. Maybe this was working for iOS photos in the older version of `react-native-image-resizer`, but not the latest version?

Reply:
It maybe a bug with `react-native-image-resizer`, but I was finding that it wasn't able to handle iOS localIdentifiers like old versions did, so I had to use `RNFS.copyAssetsFileIOS` instead, which does require you to specify and absolute output path.